src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -14,4 +26,19 @@
 public class WalletController {
 
     private final WalletService walletService;
+
+    @PutMapping("/update")
```

Comment:
Maybe we should pass `id` of the wallet to update by the endpoint? so `/update//{id}` or even just `{id}`

Reply:
That is a good question. To be honest in the task descripion it is not clearly stated how the endpoint should look like, therefore I decided to use just "/update" path without ID, because ID is included in the response body, simultaneously having the validation of provided parameters.

Reply:
Have you ever heard about "Richardson Maturity Model"? We implement Rest Architecture and this mean that we use "Richardson Maturity Model". For our project we use level 2: https://martinfowler.com/articles/richardsonMaturityModel.html#level2 . According to this cannot use verbs as an endpoint  (update is a verb). Please remove "/update". Because we have @RequestMapping("api/wallet") - for the method "updateWallet" you can use {id} as as a @PathVariable.